modules/sso.php

```diff
@@ -746,7 +865,42 @@ function allowed_redirect_hosts( $hosts ) {
 		return array_unique( $hosts );
 	}
 
-	function button( $args = array() ) {
+	/**
+	 * Builds the "Login to WordPress.com" button that is displayed on the login page as well as user profile page.
+	 *
+	 * @param  array   $args       An array of arguments to add to the SSO URL.
+	 * @param  boolean $is_primary Should the button have the `button-primary` class?
+	 * @return string              Returns the HTML markup for the button.
+	 */
+	function build_sso_button( $args = array(), $is_primary = false ) {
+		$url = $this->build_sso_button_url( $args );
+		$classes = $is_primary
+			? 'jetpack-sso button button-primary'
+			: 'jetpack-sso button';
+
+		return wp_kses(
```

Comment:
I think @eliorivero suggestion make sense if there is some reason to translate the link like if we hat a link title for example. But the way it is right now I don't think it makes much sense to do that. 

It just makes it easier for translators to break the functionality. If they miss a `>` for example.